phpstan.neon.dist

```diff
@@ -1,3 +1,5 @@
+includes:
+	- vendor/szepeviktor/phpstan-wordpress/extension.neon
```

Comment:
Make PHPStan find it automatically using `phpstan/extension-installer`.

```
composer require --dev phpstan/extension-installer
```

https://github.com/szepeviktor/phpstan-wordpress

Reply:
✅ in https://github.com/pronamic/pronamic-pay-ideal-2/pull/5/commits/5fdcb6881373646c979b3b956fa2f75a336fe451.